### ISSUE TITLE
Review our app via Google Accessibility Scanner

### ISSUE THREAD

**Issue by fm-sys** [cgeo/cgeo#10882]
There is an App called [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=de&gl=US) which can scan the UI if there are issues like bad contrasts or small buttons which might be hard to hit for some people... 

This might help us to further optimize our theming.

**Comment by fm-sys** [comment]
OK, seams like there is very much to do if we want to make c:geo accessible for handicapped people...

Mostly missing texts for screen readers and way too small touch targets. 

The question: Should we optimize c:geo according to such guidelines?

| | |
|-|-|
| ![Screenshot_20210607_145424_com google android apps accessibility auditor](https://user-images.githubusercontent.com/64581222/121020178-6948b700-c7a0-11eb-94a0-4135a9522043.jpg) | ![Screenshot_20210607_145415_com google android apps accessibility auditor](https://user-images.githubusercontent.com/64581222/121020175-68178a00-c7a0-11eb-926c-bee3951c750d.jpg) |
